cg/services/order_validation_service/rules/sample/utils.py

```diff
@@ -60,3 +62,11 @@ def get_indices_for_repeated_sample_names(order: OrderWithNonHumanSamples) -> li
         if counter.get(sample.name) > 1:
             indices.append(index)
     return indices
+
+
+def does_sample_have_an_invalid_well_format(sample: Sample) -> bool:
```

Comment:
```suggestion
def is_valid_well_format(sample: Sample) -> bool:
```

Reply:
Thanks! would be `is_invalid_well_format` in this case I will update!

Reply:
I mean, for me the `is_valid_well_format` implies that we are only evaluating the format of the well position, but it actually determines if the samples well format is relevant as well. Very minor thing but I would probably call it what you had and if needed, break out the matching with the regex into a function with that name